### DISCUSSION TITLE
ChangeBackuplocation

### DISCUSSION THREAD

**Discussion by 5u5anne** [olahallengren/sql-server-maintenance-solution#826] [General]
How can I change the backup directory to a new backup directory? 

I have changed 
DECLARE @BackupDirectory nvarchar(max)     = 'C:\backup' to 'D:\backup' 
and execute the script again, but nothing changed.  
Have I missed something?

**Comment by olahallengren** [comment]
The jobs will not be changed after they have been created. You will either have to modify the jobs or drop them and rerun the script.
